db/pre-re/item_db.conf

```diff
@@ -74203,6 +74303,33 @@ item_db: (
 
 	">
 },
+{
+	Id: 12883
+	AegisName: "Almighty"
+	Name: "Almighty"
+	Type: "IT_USABLE"
+	Buy: 20
+	Weight: 10
+	Trade: {
+		nodrop: true
+		notrade: true
+		noselltonpc: true
+		nocart: true
+		nogstorage: true
+		nomail: true
+		noauction: true
+	}
+	Script: <"
+		specialeffect(EF_BASH3D, AREA, playerattached());
+		sc_start(SC_FOOD_STR, 1800000, 10);
```

Comment:
This should use the `SC_FOOD_*_CASH` variant

Reply:
Done.